10-query/00-overview.md

```diff
@@ -11,49 +11,33 @@ permalink: /docs/query/overview
-When writing Graql queries, you simply describe **what** information you would like to retrieve, rather than **how** should it be obtained. Once we specify the target information to retrieve, the Graql query processor will take care of finding an optimal way to retrieve it.
+When writing Graql queries, we simply describe **what** information we would like to retrieve, rather than **how** should it be obtained.
+Once we specify the target information to retrieve, the Graql query processor will take care of finding an optimal way to retrieve it.
 
 **Graql is intuitive**.
-
-Graql was designed to provide a high-level query language interface with a clear and human-readable syntax. By defining high-level application-specific [schema](/docs/schema/overview),
-we effectively define our own vocabulary to talk about the domain of interest. By introduction of an explicit data model tightly reflected in the structure of the query language,
-formulating queries comes naturally as it is reminiscent of building ordinary sentences about our domain. The more tightly the schema represents our domain of interest,
-the more intuitive writing and reading Graql queries become.
+Graql was designed to provide a high-level query language interface with clear and human-readable syntax. By defining high-level application-specific [schema](/docs/schema/overview), we effectively define our own vocabulary to talk about the domain of interest. By introduction of an explicit data model tightly reflected in the structure of the query language, formulating queries comes naturally as it is reminiscent of building ordinary sentences about our domain. The more tightly the schema represents our domain of interest, the more intuitive writing and reading Graql queries become.
 
 **Graql serves as both the Data Manipulation Language (DML) as well as the Data Definition Language (DDL)**
-
-Graql is a language that provides you with a complete set of tools to perform all data-oriented tasks.
-This includes defining the schema, retrieving information as well as creating and manipulating data. 
-
+Graql is a language that provides you with a complete set of tools to perform all data-oriented tasks. This includes defining the schema, retrieving information as well as creating and manipulating data.
 
 ## The structure of a Graql query
 
-Let's look at the anatomy of a typical Match-Get query:
+Let's look at the anatomy of a typical Get query:
 
 ![Query structure](/docs/images/query/query-structure.png)
 
-The important parts were marked with numbers, as a result we have:
+The important parts were marked with numbers. As a result, we have:
+
+1. The **`match` keyword** that is followed by the pattern to look for within the knowledge graph. The `match` keyword and the following pattern, commonly known as the _match clause_, is always accompanied by a _query keyword_. In the example above, that query keyword happens to be `get`. `match` may also be accompanied with, `insert` and `delete`.
```

Comment:
It should say **query keyword**. Match keyword is a special case in the provided example, it doesn't have to be match.

Query keyword specifies the query type and subsequently the nature of the operation we want the query to carry out.
In that way we can differentiate between data manipulation and update operations(`match`, `compute`), as well as data definition operations (`define`, `insert`).